src/apiProxy/api-proxy.js

```diff
@@ -33,30 +33,26 @@ import {
  * query-specific options.
  *
  * @param {Request} request Hapi request object
- * @param {Object} baseUrl API server base URL for all API requests
  * @return Array$ contains all API responses corresponding to the provided queries
  */
-const apiProxy$ = ({ API_TIMEOUT=8000, baseUrl='', duotoneUrls={} }) => {
-
-	return request => {
-		request.log(['api', 'info'], 'Parsing api endpoint request');
-		// 1. get the queries and the 'universal' `externalRequestOpts` from the request
-		const { queries, externalRequestOpts } = parseRequest(request, baseUrl);
-
-		// 2. curry a function that uses `externalRequestOpts` as a base from which
-		// to build the query-specific API request options object
-		const queryToRequestOpts = buildRequestArgs(externalRequestOpts);
-
-		request.log(['api', 'info'], JSON.stringify(queries));
-		// 3. map the queries onto an array of api request observables
-		const apiRequests$ = queries
-			.map(queryToRequestOpts)
-			.map((opts, i) => ([opts, queries[i]]))  // zip the query back into the opts
-			.map(makeApiRequest$(request, API_TIMEOUT, duotoneUrls));
-
-		// 4. zip them together to send them parallel and return responses in order
-		return Rx.Observable.zip(...apiRequests$);
-	};
+const apiProxy$ = request => {
+	request.log(['api', 'info'], 'Parsing api endpoint request');
```

Comment:
this is primarily an indentation change because the config values no longer need to be threaded through this function into `parseRequest` and `makeApiRequest$`